enterprise/frontend/src/embedding-sdk/cli/constants/messages.ts

```diff
@@ -53,3 +60,16 @@ export const getMetabaseInstanceSetupCompleteMessage = (instanceUrl: string) =>
 
   Read more: https://www.metabase.com/docs/latest/installation-and-operation/information-collection
 `;
+
+export const getExpressServerGeneratedMessage = (filePath: string) => {
+  const NPM_INSTALL_DEPS_COMMAND = chalk.blue(
+    "npm install express express-session jsonwebtoken cors",
+  );
+
+  return `
+  Generated an example Express.js server in "${filePath}".
+  Add the dependencies with "${NPM_INSTALL_DEPS_COMMAND}"
```

Comment:
I don't want to automatically add this to their package.json dependency, because some projects separate frontend and backend into workspaces, and some combines them. Best to let them do this manually.